### ISSUE TITLE
2023R1: Investigate AlmaLinux 9

### ISSUE THREAD

**Issue by bieryAtFnal** [bieryAtFnal/3proclistrev#10]
## Introduction
The AlmaLinux9 OS has been requested for a couple of teststand environments, and it will be needed at some point at EHN1 and other DAQ environments.  In addition, there is a desire to switch over to it sooner rather than later so that this disruption doesn't affect data taking later in the year.

We are considering what level of commitment we should make for rolling out this OS version for the April 2023 release.  Initial tests have shown that the _dunedaq_ software is already buildable on AL9, but exhaustive testing has not yet been done with real electronics hardware.  Such testing is proposed to happen between 30-Jan and 17-Feb.  Once the testing is done, decisions will be made on how to proceed.

## Planned Deliverables

1. Investigate whether existing _dunedaq_ software can successfully be run with AL9. This should include testing with FELIX, DPDK, timing electronics, AVX2, etc.

   <details>
      <summary>Narrative</summary>
      to-do: fill in more details as they become known
   </details>

   <details>
      <summary>Observables</summary>
      One or more computers should be converted to AL9, the software should be installed and built, and tests with various data-taking scenarios should be run.  The observables should be the following:

      1. successful test run with real FELIX (e.g. something along the lines of daq-systemtest/integtest/felix_emu_wib2_test.py
      2. etc.
   </details>

   <details>
      <summary>Suggested testing</summary>

      1. confirm that all of the observables are present
      3. (add more validation/acceptance tests here)

   </details>

## Deferred/Excluded Deliverables

None so far.

## Reference Material

None so far.
